highfidelity/hifi-content
# Adding a mirror with a button as a Blueprint

Test Plan:

1. Import the following marketplace entity by URL: http://mpassets.highfidelity.com/40b93ca0-c08f-42a8-9034-462f8bfa5fcb-v1/mirror_with_button.json

2. Confirm that the mirror appears, and turns on/off when the button is clicked (desktop) and far-triggered (HMD)

Note: there appears to be a bug that I'm looking into where the first several times you import items from marketplace via the 'Import Entities from URL' menu, the scripts do not run until you open the Create menu (or wait quite a long time for them to load). That issue is not specific to this piece of content, but if you hit it in testing, it will be a valuable info point.
